.ebextensions/05_loadbalancer-terminatehttps.config

```diff
@@ -1,7 +1,7 @@
 option_settings:
   aws:elb:listener:443:
     ListenerProtocol: HTTPS
-    SSLCertificateId: arn:aws:acm:us-east-1:224280085904:certificate/81061d9f-0a08-4440-bdc0-016e4d40b423
+    SSLCertificateId: arn:aws:acm:us-east-1:946183545209:certificate/aee40806-85d9-4e4f-973d-4430105a0369
```

Comment:
Since we changed base branch, you probably want to make sure this aws profile is still correct.

Reply:
This is set back to `nypl-sandbox` credentials right now, will set to `nypl-digital-dev` credentials before merging into qa

Reply:
Cool thanks!